shared/request_caching.go

```diff
@@ -75,8 +77,13 @@ type cachingHandler struct {
 	cache       ReadWritable
 	isCacheable func(*http.Request) bool
 	getCacheKey func(*http.Request) interface{}
+	shouldCache func(int) bool
 }
 
+func defaultIsCacheable(*http.Request) bool          { return true }
```

Comment:
So, in another in-flight PR I've made this public and actually called this 'AlwaysCachable':
https://github.com/web-platform-tests/wpt.fyi/pull/674/files#diff-6c9911785ad4f162692fbb1bcc25984aR149